freqtrade/freqtradebot.py

```diff
@@ -362,13 +364,56 @@ def process_maybe_execute_sell(self, trade: Trade) -> bool:
         if trade.open_order_id:
             # Update trade with order values
             logger.info('Found open order for %s', trade)
-            trade.update(exchange.get_order(trade.open_order_id, trade.pair))
+            order = exchange.get_order(trade.open_order_id, trade.pair)
+            # Try update amount (binance-fix)
+            try:
+                # Only run for closed trades
+                if trade.fee_open != 0 and not (order['status'] == 'open'):
+                    new_amount = self.get_real_amount(trade)
+                    # This may break if a exchange applies no fee (which appears highly unlikely)
+                    if order['amount'] != new_amount:
+                        logger.info("Updating amount for Trade {} from {} to {}".format(
```

Comment:
The amount update for trade happens in: https://github.com/gcarq/freqtrade/blob/feat/objectify-ccxt/freqtrade/persistence.py#L123. I think we should either log it their or rephrase this log msg

Reply:
The reason for this log msg is to somehow note/log that this was changed due to this logic (amount-updates could also happen for other reasons i assume, as the code was already there). it's easy to grep for "Updating" through the logs, and find trades where this applied (at least that was my thought-process).

I'll try and phrase it clearer.